src/functions/template-tags/organizer.php

```diff
@@ -332,19 +347,25 @@ function tribe_get_organizer_website_url( $postId = null ) {
 	function tribe_get_organizer_website_link( $post_id = null, $label = null ) {
 		$post_id = tribe_get_organizer_id( $post_id );
 		$url     = tribe_get_event_meta( $post_id, '_OrganizerWebsite', true );
+		$target = apply_filters( 'tribe_get_event_organizer_link_target', '_self' );
+		$rel    = ( '_blank' === $target  ) ? 'noopener noreferrer' : 'external';
```

Comment:
* :warning: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 4 spaces
* :no_entry_sign: Expected 1 space before close parenthesis; 2 found

<sub>hash: c05f003d0e3f70fb8dd54dcaff816f99</sub>